### PR TITLE
User display helper

### PR DESCRIPTION
**Pull request by lowjoel**
To standardise how users are displayed across the application.

### CODE REVIEW

File: spec/helpers/application_helper_spec.rb [hunk 1]
```diff
@@ -28,4 +28,37 @@
...
     end
   end
+
+  describe 'user display helper' do
+    describe '#display_user' do
+      let(:user) { build(:user) }
+      subject { helper.display_user(user) }
+
+      it do
+        pending 'Name'
```
Comment (jsyeo) [inline spec/helpers/application_helper_spec.rb]: pending?

Comment (lowjoel) [inline spec/helpers/application_helper_spec.rb]: Need to merge #68 for this...

Comment (lowjoel) [inline spec/helpers/application_helper_spec.rb]: Then another PR to change the pending spec afterward.

Comment (lowjoel) [inline spec/helpers/application_helper_spec.rb]: which I just did, let me rebase.

**Pull request merged**